charts/csm-replication/templates/controller.yaml

```diff
@@ -0,0 +1,297 @@
+apiVersion: v1
```

Comment:
Similar concern as before. This is [a duplicate of a file in the csm-replication repository](https://github.com/dell/csm-replication/blob/main/deploy/controller.yaml).

Reply:
We need these charts to be placed in helm repo to install it using CSM installation wizard